jbpm-wb-kie-server/jbpm-wb-kie-server-backend/src/main/java/org/jbpm/workbench/ks/integration/KieServerDataSetProvider.java

```diff
@@ -391,9 +398,13 @@ protected void handleDataSetGroup(DataSetDef def, DataSetGroup dataSetGroup, Lis
                                                     null,
                                                     Arrays.asList(groupFunction.getColumnId())));
                     extraColumns.add(new DataColumnImpl(groupFunction.getSourceId(),
-                                                        def.getColumnById(groupFunction.getColumnId()).getColumnType()));
+                                                        getGroupFunctionColumnType(def.getColumnById(groupFunction.getColumnId()).getColumnType())));
                 }
```

Comment:
@cristianonicolai I think this DATE to LABEL "fix" must only be applied to the group column:

...
        extraColumns.add(new DataColumnImpl(groupFunction.getSourceId(),                                                         getGroupFunctionColumnType(columnGroup, groupFunction)));
      }
   }
}

 protected ColumnType getGroupFunctionColumnType(final ColumnGroup columnGroup, final GroupFunction groupFunction) {
       ColumnType type = def.getColumnById(groupFunction.getColumnId()).getColumnType();
        return type == ColumnType.DATE && columnGroup.getSourceId().equals(groupFunction.getSourceId()) ? ColumnType.LABEL : type;
 }

wdyt?

Reply:
This makes sense to me so +1